src/main/resources/tools/developer/jwtConfiguration.jsp

```diff
@@ -22,7 +22,7 @@
 
 <html>
 <head>
-    <script src="${pageContext.request.contextPath}/modules/security-filter-tools/javascript/apps/security-filter-jwt.js" ></script>
+    <script src="${pageContext.request.contextPath}/modules/security-filter-tools/javascript/apps/jahia.bundle.js" ></script>
```

Comment:
Does it work ? The jahia.bundle.js is built with a DLL and cannot be called by itself.  We don't need this JSP anyway, so you can remove it

Reply:
yes the JSP isn't needed anymore, it's not used